cmd/buildx/main.go

```diff
@@ -112,11 +111,6 @@ func main() {
 	var ebr *desktop.ErrorWithBuildRef
 	if errors.As(err, &ebr) {
 		ebr.Print(cmd.Err())
-	} else {
-		var be *controllererrors.BuildError
-		if errors.As(err, &be) {
-			be.PrintBuildDetails(cmd.Err())
```

Comment:
Can you explain bit what is going on here and commands/build and why this needed anymore. Where is the replacement `PrintBuildDetails`.

Reply:
This section was previously needed because `desktop.ErrorWithBuildRef` wasn't serializable. So this type was used as a wrapper for `desktop.ErrorWithBuildRef`. `PrintBuildDetails` will just create a `desktop.ErrorWithBuildRef` and then call `Print` so it's equivalent to the above.